Dockerfile.frontend

```
@@ -7,7 +7,7 @@
 # when the dependencies change.
 
 # Start with the official Node.js image to install dependencies
-FROM node:16.20.0-alpine AS modules
+FROM node:20.11.1 AS modules
```

We should probably continue to use the alpine image here -- it's much smaller than the default image! (`node:20.11.1` is 380.62 MB, while `20.11.1-alpine` is a mere 45.72 MB!)

```suggestion
FROM node:20.11.1-alpine AS modules
```